P1928/wording.tex

```diff
@@ -1345,6 +1345,8 @@
 \end{itemdecl}
 
 \begin{itemdescr}
+  \pnum\constraints
+  \tcode{totally_ordered<T>} is \tcode{true}
```

Comment:
`\tcode{T} satisfies \tcode{totally_ordered}.`

And then please add an empty line before the next `\pnum`

I'm not sure whether we also want
```tex
\pnum\expects
\tcode{T} models \tcode{totally_ordered}.
```
NaNs don't *model* the concept. So I believe this precondition would say that passing NaNs triggers UB.

Reply:
Fixed